src/checks/leading_space.rs

```diff
@@ -15,8 +17,10 @@ impl Default for LeadingSpaceChecker {
 
 impl Check for LeadingSpaceChecker {
     fn run(&mut self, line: LineEntry) -> Option<Warning> {
-        // FIXME: Doesn't check a tab character
-        if line.raw_string.starts_with(' ') {
+        // Allow Uppercase, Lowercase and # as first character
+        let re = Regex::new(r"^[A-Za-z#]").unwrap();
+
+        if !re.is_match(&line.raw_string) {
```

Comment:
I think we can check it easier, without using the `regex` crate:
```rust
if !line.raw_string.starts_with(|c: char| c.is_alphabetic() || c == '_') {
    // return Warning
} else {
    // return None
}
```

@mstruebing What do you think about the example above?